index.rst

```diff
@@ -1,63 +1,137 @@
+* Always add units to physical quantities.
+* Avoid differentiating data ranges with colors. Try to use different line and marker styles. If colors are needed, a recommendation is to upload your plots to `this page <https://www.color-blindness.com/coblis-color-blindness-simulator>`_
+* Include error bars
+* Highlight requirements or areas in which the viewer has to focus on
+* Where applicable provide histogrammed versions of the results and calculate a mean and RMS, or equivalently a Gaussian fit
 
-- Passive voice?
-- "We" perspective?
+An example is shown in :numref:`label` :
 
-**Reason:**
+.. _label:
+.. figure:: /_static/azel.png
+   :name: fig-azel
+
+   Distribution of slew events, including those that failed the test, in azimuth and elevation, for dayObs 
+   20232012.
+
+Discussion
+==========
+A detailed discussion could have its own section after the Results, but is not always necessary. This is particularly useful when the amount of tables and/or plots require an overall evaluation to be explained in a single section. 
+
+Conclusions
+===========
+Conclusions should include a clear cut status of the resolution of the goal the technote was designed
+to address. In case of requirement verification, this would include PASS/FAIL assessments for each.
+
+If a clear resolution is not found at this time, it is advisable to add a path to said resolution
+or further testing, as sometimes technotes are only meant to provide a snapshot of the situation
+they are describing. 
+
+This section should not be too long to provide a quick 'single-glance' summary, that together with the abstract, would provide a complete sself contained information piece on the issue at hand.
+
+*Example: Using star tracker data from camera XX on the soak tests performed on observation day XXXX-YY-ZZ we have been able to verify requirement RRR, in mostly all azimuth-elevation combinations. However, in 60% of the cases were pointing was to an elevation below 30 degrees, the offset was beyond the requirement, reaching almost 8 arcseconds.*
+
+Related Documentation
+=====================
+Documentation references with links to other relevant technotes or Rubin documentation, including other technical bibliography such as papers, conference proceedings, where appropriate.
+
+*Example:*
+
+`LTS-88 <https://docushare.lsst.org/docushare/dsweb/Get/LTS-88/LTS-88.pdf>`_ *M1M3 Mirror Support Design Requirement Document*
+
+
+Appendix
+========
+
+Technote Writing Guide
+----------------------
 
-**Example:**
+In order to start a technote, as well as some useful tips for using reStructured text, please refer to this `presentation <https://github.com/lsst-sitcom/sitcomtn-102/files/14638694/2020-10-14.Documentation.Ops.Bootcamp.pdf>`_ by the Rubin documentation team. ReStructured text supports LaTeX-style math using the 'math' environment and inverted commas: \:math\:\`x^2+y^2=z^2\` will translate into :math:`x^2+y^2=z^2`.
```

Comment:
New line for "ReStructured text supports LaTeX-style math using the 'math' environment and inverted commas: \:math\:\`x^2+y^2=z^2\` will translate into :math:`x^2+y^2=z^2`."